packages/cli/src/commands/init/init.ts

```diff
@@ -202,13 +217,32 @@ async function createProject(
     projectName,
     templateUri,
     npm: options.npm,
+    bun: options.bun,
     directory,
     projectTitle: options.title,
     skipInstall: options.skipInstall,
     packageName: options.packageName,
   });
 }
 
+function resolvePackageManager() {
+  const userAgent = process.env.npm_config_user_agent;
```

Comment:
this is more "override package manager based on user config", right? maybe let's rename this fn accordingly